public/app/core/logs_model.ts

```diff
@@ -480,3 +466,48 @@ function getIdField(fieldCache: FieldCache): FieldWithIndex | undefined {
   }
   return undefined;
 }
+
+// Format timeSpan (in sec) to string used in log's meta info
+function formatTimeSpanToTimeString(timeSpanSec: number): string {
+  const h = Math.floor(timeSpanSec / 60 / 60);
+  const m = Math.floor(timeSpanSec / 60) - h * 60;
+  const s = Number((timeSpanSec % 60).toFixed());
+  let formattedH = h ? h + 'h' : '';
+  let formattedM = m ? m + 'min' : '';
+  let formattedS = s ? s + 'sec' : '';
+
+  formattedH && formattedM ? (formattedH = formattedH + ' ') : (formattedH = formattedH);
+  (formattedM || formattedH) && formattedS ? (formattedM = formattedM + ' ') : (formattedM = formattedM);
+
+  return formattedH + formattedM + formattedS || 'less than 1sec';
+}
+
+// Used to add additional information to Line limit meta info
+function adjustMetaInfo(logsModel: LogsModel, timeSpanOfLogs?: number, timeSpanOfRange?: number): LogsMetaItem[] {
+  let logsModelMeta = [...logsModel.meta!];
+
+  const limitIndex = logsModelMeta.findIndex((meta) => meta.label === 'Line limit');
```

Comment:
Is there something else we could use to filter? Like an id? Or maybe "Line limit" could be const so nobody changes the label without breaking the filter?

Reply:
Yeah, we can create const form it.